sample-queries.csv

```diff
@@ -160,6 +160,16 @@ Excel,GET,used range in worksheet,https://graph.microsoft.com/v1.0/me/drive/item
 Excel,GET,tables in worksheet,https://graph.microsoft.com/v1.0/me/drive/items/{drive-item-id}/workbook/worksheets('Sheet1')/tables,,"This query requires a driveItem id.  To find the ID of the driveItem that corresponds to an Excel Workbook, you can run: GET https://graph.microsoft.com/v1.0/me/drive/root/search(q='.xlsx')?select=name,id,webUrl.",https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/worksheet_list_tables,AAD,,,
 Excel,GET,charts in worksheet,https://graph.microsoft.com/v1.0/me/drive/items/{drive-item-id}/workbook/worksheets('Sheet1')/charts,,"This query requires a driveItem id.  To find the ID of the driveItem that corresponds to an Excel Workbook, you can run: GET https://graph.microsoft.com/v1.0/me/drive/root/search(q='.xlsx')?select=name,id,webUrl.",https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/worksheet_list_charts,AAD,,,
 Planner,GET,all Planner plans associated with a group,https://graph.microsoft.com/v1.0/groups/{group-id-with-plan}/planner/plans,,"This query requires a group id.  To find the ID of a group you belong to, you can run: GET https://graph.microsoft.com/v1.0/me/memberOf",https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/plannergroup_list_plans,AAD,,,
+Planner,GET,Planner plan,https://graph.microsoft.com/v1.0/planner/plans/{plan-id},,This query requires a plan id.  To find the ID of the plan you can run: GET https://graph.microsoft.com/v1.0/me/groups/{group-id}/plans.,https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/plannerplan_get,AAD,,,
+Planner,PATCH,update a Planner Plan,https://graph.microsoft.com/v1.0/planner/plans/{plan-id},Group.ReadWrite.All,This query requires a Plan id and value of @odata.etag for a selected task.  To find the ID of the Plan you can run: GET https://graph.microsoft.com/v1.0/me/planner/tasks and then use retrieved {plan-id} to find @odata.etag run: GET https://graph.microsoft.com/v1.0/planner/plans/{plan-id},https://developer.microsoft.com/en-us/graph/docs/api-reference/v1.0/api/plannerplan_update,AAD,"{
```

Comment:
I would revise the tip to read:
This query requires a Plan ID and value of @odata.etag for a selected task.  To find the ID of the Plan, you can run: GET https://graph.microsoft.com/v1.0/me/planner/tasks.  To get the @odata.etag, run: GET https://graph.microsoft.com/v1.0/planner/plans/{plan-id}